fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/models/FedoraResource.java

```diff
@@ -181,6 +196,7 @@
      *
      * @param type the type to add
      */
+    @Deprecated
```

Comment:
Is this to assist in finding usages? 'Cause otherwise I'd say just remove them if we don't need them.

Reply:
I've gone ahead and removed it plus the one usage of it. I also deprecated all the modification methods in `Binary`, but opted to create a ticket for cleaning them up since they are referenced in a couple places https://jira.lyrasis.org/browse/FCREPO-3146